modules/_labs/pagination/react/lib/Pagination.tsx

```diff
@@ -123,7 +123,9 @@ const Pagination = (props: PaginationProps) => {
           />
         </ButtonsContainer>
         {showGoTo && <GoTo onSubmit={onPageChange} max={numPages} label={goToLabel} />}
-        {showLabel && <StyledLabel>{customLabel(labelFrom, labelTo, total)}</StyledLabel>}
+        {showLabel && (
+          <StyledLabel aria-live="polite">{customLabel(labelFrom, labelTo, total)}</StyledLabel>
+        )}
```

Comment:
Is this an okay opt-in? If the label is shown, it will get announced to screenreaders on update. That does allow an application developer to disable only by providing their own label...

If this is an acceptable solution, I think we should add something to the README.md about this accessibility feature so at least we let people know it is happening and it could possibly double announce depending on what the rest of the application does when the page value changes.